sdk/python/packages/flet-core/src/flet_core/control.py

```diff
@@ -198,11 +197,13 @@ def page(self, page):
         self.__page = page
 
     # uid
+
```

Comment:
Seems like your formatter is still on. 😅

Reply:
I'll fix that